Dockerfile

```diff
@@ -36,6 +36,9 @@ ENV INBUCKET_STORAGE_PARAMS path:/storage
 ENV INBUCKET_STORAGE_RETENTIONPERIOD 72h
 ENV INBUCKET_STORAGE_MAILBOXMSGCAP 300
 
+# Healthcheck
+HEALTHCHECK --interval=5s --timeout=5s --retries=3 CMD /bin/sh -c 'wget localhost:$(echo ${INBUCKET_WEB_ADDR:-0.0.0.0:9000}|cut -d: -f2) -q -O - > /dev/null 2>&1'
```

Comment:
I haven't used HEALTHCHECK before, but would it make sense to eliminate `2>&1`?  Given the docs state:

> To help debug failing probes, any output text (UTF-8 encoded) that the command writes on stdout or stderr will be stored in the health status and can be queried with docker inspect.

I know stdout will have a lot of HTML output, but stderr might have something useful if there was an actual failure?